_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,124 @@
+membrane biophysics through molecular dynamics simulations coupled
+with enhanced sampling techniques.  Hugo can be found on github as
+@hmacdope and on twitter as [@hugomacdermott][hmacdope-twitter]. When
+not hard at work Hugo can be found running or mountain biking in the
+Canberra hills.
+
+Through GSoC Hugo aims to bring the TNG next generation trajectory
+format to the simulation community and he will document his experience at
+[his "Biophysics Bonanza" blog][hmacdope-blog].
+
+
+## Cédric Bouysset: From RDKit to the Universe and back
+
+<img
+src="https://cbouy.github.io/assets/img/photo-CV.jpeg"
+title="Cédric Bouysset" alt="Cédric Bouysset"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+The aim of the [RDKit interoperability
+project](https://summerofcode.withgoogle.com/projects/#6750913248624640)
+is to give MDAnalysis the ability to use [RDKit][]’s `Chem.Mol`
+structure as an input to an MDAnalysis’ `Universe`, but also to
+convert a `Universe` or `AtomGroup` to an RDKit `molecule`. RDKit is
+one of the most complete and one of the most commonly used
+chemoinformatics package, yet it lacks file readers for formats
+typically encountered in MD simulations. @cbouy will implement in
+MDAnalysis the ability to switch back and forth between a `Universe`
+and an RDKit `molecule` to perform typical chemoinformatics
+calculations and so add a lot of value to both packages.
+
+Cédric is a PhD student at the Institute of Chemistry of Nice, in the
+south of France. His research project tackles taste perception (and
+sometimes olfaction) using computational approaches. His day-to-day
+work includes homology modeling of bitter taste receptors, building
+machine-learning models to predict taste modalities from chemical
+structures, writing Python or Bash scripts, and a bit of teaching.
+
+Cédric will describe his progress in [his blog][cbouy-blog].
+
+
+## Yuxuan Zhuang: Serialize Universes for parallel
+
+<img
+src="https://i0.wp.com/www.biophysics.se/wp-content/uploads/2018/06/IMG_4767.jpg"
+title="Yuxuan Zhuang" alt="Yuxuan Zhuang"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+MDAnalysis is a flexible and relatively fast framework for complex
+analysis tasks in molecular dynamics (MD) simulations. To achieve a
+flawless implementation of parallelism, @yuxuanzhuang will implement
+[serialization support for
+`Universe`](https://summerofcode.withgoogle.com/projects/#5812065073102848),
+the core of MDAnalysis. Furthermore, he will adapt this new
+serialization functionality into Dask, multiprocessing, or
+MPI. Additionally, he will run tests, write documentation, and run
+benchmarks.
+
+Yuxuan is a first-year PhD student at Stockholm University. He mainly
+works on understanding ion channels from MD simulations in Prof. Erik
+Lindahl’s group. My daily workflow involves setting up simulations,
```

Comment:
Relating to the `My` starting sentence. The text here jumps from third person to first person. Probably worth changing?

Reply:
Fixed in 05b70f9 and added some of the changes in the email @yuxuanzhuang sent